src/renderer/components/description/description.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import './description.scss';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  selectCurrentUser,
+} from '../../../shared/slices/currentUserSlice';
+import {
+  selectDefaultDirPath,
+  setDefaultDirPath,
+} from '../../../shared/slices/configurationSlice';
+import findLocalPublications from '../../../main/node/file-manager';
```

Comment:
you do `export default {findLocalPublications, isDirectory, isPublication}` but import this whole object as `findLocalPublications` here? are you sure?

Reply:
https://github.com/karczewskiPiotr/vnlab-tool/pull/14/commits/d88c5b71a899d8138b92c372c455365d5354e667
I think this commit changes affect this. Did you test this code after this change?

Reply:
same as above, the description file shouldn't be here. I'm just gonna delete it